### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by Big-Iron-Cheems**
## Type of change

- [ ] Bug fix
- [X] New feature

## Description

Updated the issue templates to include a dedicated Crash template.

## Related issues

None

# How Has This Been Tested?

No testing required.

# Checklist:

- [X] My code follows the style guidelines of this project.
- [X] I have added comments to my code in more complex areas.
- [X] I have tested the code in both development and production environments.

### CODE REVIEW

**Comment by Paddyk45** [comment]
Is it possible to make steps to reproduce without bash highlighting? If yes, could you add that too

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -8,32 +8,35 @@ body:
...
       required: true
   - type: textarea
     id: reproducing
     attributes:
       label: Steps to reproduce
       description: How do you trigger this bug?
+      placeholder: |
+          1. I did this thing;
+          2. Then I did this other thing, which caused the bug.
       render: bash
```
Comment (Paddyk45) [inline .github/ISSUE_TEMPLATE/bug.yml:25]: Remove this

Comment (Paddyk45) [inline .github/ISSUE_TEMPLATE/bug.yml:25]: Remove this

Comment (Paddyk45) [reply .github/ISSUE_TEMPLATE/bug.yml:25]: I give up trying to highlight the line

**Comment by Big-Iron-Cheems** [comment]
I'll do it once I get back to my PC.

**Pull request merged**